phenol-core/src/main/java/org/monarchinitiative/phenol/ontology/data/ImmutableOntology.java

```diff
@@ -258,25 +264,28 @@ public static Builder builder() {
   }
 
   public static class Builder {
-    private Map<String, String> metaInfo = new LinkedHashMap<>();
-    private Collection<Term> terms = new ArrayList<>();
-    private Collection<Relationship> relationships = new ArrayList<>();
+    private final SortedMap<String, String> metaInfo = new TreeMap<>();
+    private final Collection<Term> terms = new ArrayList<>();
+    private final Collection<Relationship> relationships = new ArrayList<>();
+
+    private Builder() {
+    }
 
     public Builder metaInfo(Map<String, String> metaInfo) {
       Objects.requireNonNull(metaInfo);
-      this.metaInfo = new LinkedHashMap<>(metaInfo);
+      this.metaInfo.putAll(metaInfo);
       return this;
     }
 
     public Builder terms(Collection<Term> terms) {
       Objects.requireNonNull(terms);
-      this.terms = new ArrayList<>(terms);
+      this.terms.addAll(terms);
       return this;
     }
 
     public Builder relationships(Collection<Relationship> relationships) {
       Objects.requireNonNull(relationships);
-      this.relationships = new ArrayList<>(relationships);
+      this.relationships.addAll(relationships);
```

Comment:
Don't do this - calling this twice now changes the behaviour and isn't in line with the method name. See above.